src/actions/loadData.js

```diff
@@ -240,6 +240,47 @@ export const loadSecondTree = (secondTreeUrl, firstTreeUrl) => async (dispatch,
 };
 
 
+const loadMultipleBlocks = (blocks) => async (dispatch) => {
+  const jsons = {};
+  const namesMainTree = [];
+  const namesTreeToo = [];
+
+  for (let i = 0; i < blocks.length; i++) {
+    namesMainTree[i] = blocks[i].dataset.split(":")[0];
```

Comment:
I think it'd be better to use `collectDatasetFetchUrls` rather than splitting on `:` here

Reply:
Good point!